src/MCPServer/lib/RPCServer.py

```diff
@@ -113,6 +113,8 @@ def package_create_handler(*args, **kwargs):
         payload.get('accession'),
         payload.get('path'),
         payload.get('metadata_set_id'),
+        payload.get('auto_approve'),
+        payload.get('wait_until_complete'),
```

Comment:
When `auto_approve=False` and `wait_until_complete=True` we would be effectively implementing the same behaviour of the old `start_transfer` endpoint. So I'm hoping with some further development we can have the `start_transfer` view updated so it delegates on MCPServer's new package.py module instead of the old `filesystem_ajax` Django app.